docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
+
+Both the [command]`{bin}` binary and the {prod} virtual machine can be configured using the [command]`{bin} config` command.
+The [command]`{bin} config` command requires a subcommand to act on the configuration.
+The available subcommands are `get`, `set,` `unset`, and `view`.
```

Comment:
Why do you use the [command] prefix for `{bin} config`, but not for these subcommands?

Reply:
Succinctly, because `crc config` is a command -- `get`, `set`, `unset`, and `view` are subcommands that cannot be directly run.